src/services/dashboards/modules/dashboard-label/DashboardLabels.vue

```diff
@@ -32,25 +32,27 @@
 <script setup lang="ts">
 // this vOnClickOutside is using !! Please do not remove.
 import { vOnClickOutside } from '@vueuse/components';
-import { reactive } from 'vue';
+import { computed, reactive } from 'vue';
 
 import { PIconButton, PLabel, PTextInput } from '@spaceone/design-system';
 
 
 interface Props {
-    labelList: Array<{label: string}>;
+    labelList: Array<string>;
     editable?: boolean;
 }
 const props = defineProps<Props>();
 const emit = defineEmits(['update:labelList']);
 
 const state = reactive({
-    labelList: [...props.labelList],
+    labelList: computed({
+        get: () => props.labelList,
+        set(val) { emit('update:labelList', val); },
+    }),
```

Comment:
praise: Oh! you used it ! 😄

Reply:
question: Is the type of this computed ref auto detected? 🤔

Reply:
Yes it is. But I'll add type for readablity